docs/walkthrough/using-callbacks.md

```diff
@@ -0,0 +1,126 @@
+(using-callbacks)=
+# Using Callbacks
+
+Callbacks are a way of adding additional methods to the finetuning process. The methods are executed when certain events occur and there are several callback classes, each serving a different function by providing different methods for different events.
+A run can be assigned multiple callbacks using the optional `callbacks` parameter when it is created.
+
+```python
+run = finetuner.fit(

```

Comment:
add a line above to import the callbacks, from finetuner.callback import ...

Reply:
added now